src/uv-common.c

```diff
@@ -67,6 +67,14 @@ static uv__allocator_t uv__allocator = {
   free,
 };
 
+char* uv__strdup(const char* s) {
```

Comment:
@yichoi My motivation is/was to keep libtuv as close to its origin as possible (just like @glistening mentioned). I've added now an `__APPLE__` guard around the function implementation in uv-common.c. (While I was at it, I've added the same guard to uv-common.h, because the function declaration was misleadingly available to any platforms, unlike the implementation.)